parser/parser.go

```diff
@@ -119,7 +119,7 @@ func NewFromPath(path string) (Parser, error) {
 		return New(YAML)
 	}
 
-	if fileExtension == "tf" || fileExtension == "tfvars" {
+	if fileName == "terragrunt.hcl" || fileExtension == "tf" || fileExtension == "tfvars" {
```

Comment:
regardless of the build errors, wouldn't we want to extend it to all hcl files?

Maybe with something like
```go
if fileExtension == "hcl" || fileExtension == "tf" || fileExtension == "tfvars" {
```

https://github.com/search?q=extension%3Ahcl&type=Code&ref=advsearch&l=&l=

Reply:
@caphrim007 we do have that distinction, but peeking at the codebase it doesn't even look like we support `.hcl` at all so I think looking for the `.hcl` file extension makes sense here.

Reply:
@jpreese @boranx ok, that's fine. I've updated the PR. please review at your leisure.